posthog/api/decide.py

```diff
@@ -369,6 +370,7 @@ def get_feature_flags_response_or_body(
         if token in limited_tokens_flags:
             return {
                 "quotaLimited": ["feature_flags"],
+                "flags": {},
```

Comment:
Hmm, seeing this, I actually think I might've screwed up when adding this base case in because it doesn't really take into account the API version – it doesn't really matter than much what the response is because the SDKs tend to handle it and all the data is missing anyway, but for a purist perspective I feel like we should return different empty responses based on the version.  Thoughts?

Reply:
Yeah, I agree. However, with the impending move to `/flags`, I'd rather we invest in The Right Thing™️ over there and not worry about it too much here, if that's ok.

Reply:
I changed my mind. I'll do as Spike Lee wanted us to do.

Reply:
Looks like we already had bugs in that there are places where we return `"featureFlags": []` in short-circuited paths without regard to API version. I'll fix them up.